src/record/record.js

```diff
@@ -449,7 +446,16 @@ Record.prototype._applyUpdate = function( message ) {
  */
 Record.prototype._onRead = function( message ) {
 	this.version = parseInt( message.data[ 1 ], 10 );
-	this._applyChange( jsonPath.set( this._$data, undefined, JSON.parse( message.data[ 2 ] ) ) );
+
+	this._$data = JSON.parse( message.data[ 2 ] );
```

Comment:
The problem with the old behaviour is that:

1) Subscribe to path name
2) Get read -> {}
3) name triggered with undefined, THIS IS NOT TRUE?